julia/LibCEED.jl/src/generated/libceed_bindings.jl

```diff
@@ -1366,6 +1430,10 @@ end
 
 const CeedInt_FMT = "d"
 
+const CeedSize_FMT = "td"
+
+const CeedInt8_FMT = "d"
```

Comment:
```suggestion
const CeedInt_FMT = "d"
const CeedSize_FMT = "td"
const CeedInt8_FMT = "d"
```

Reply:
This file is automatically generated with `LibCEED.jl/gen/generator.jl`, do you actually want to make manual adjustments?

Reply:
huh, odd that it vomited those on separate lines. Nevermind